### PR TITLE
Reducing docker size image and copying all sub folders

### PR DESCRIPTION
**Pull request by palaniappanrm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,14 @@
-FROM node:13
+FROM node:13-slim
 
 EXPOSE 2000
 ENV REQUIRE_HTTPS=false
-
+
```
Comment (dperelman) [inline Dockerfile:5]: Accidentally added whitespace.

Comment (dperelman) [reply Dockerfile:5]: ... I'll just make a commit fixing this after merging the PR.

**Comment by dperelman** [comment]
Thank you for your contribution. I thought the Docker image seemed overly heavyweight, but I don't know much about Docker.

**Comment by palaniappanrm** [comment]
@dperelman 
No problem and you have done so much improvement over what has been originally done by Joop. Congrats for that.

**Pull request merged**